test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresViaXml.cs

```diff
@@ -25,15 +25,17 @@ class RequiresViaXml
 		// The second attribute is added through link attribute XML
 		[RequiresUnreferencedCode ("Message for --MethodWithDuplicateRequiresAttribute--")]
 		[ExpectedWarning ("IL2027", "RequiresUnreferencedCodeAttribute", nameof (MethodWithDuplicateRequiresAttribute), ProducedBy = ProducedBy.Trimmer)]
-		[ExpectedWarning ("IL2027", "RequiresUnreferencedCodeAttribute", nameof (MethodWithDuplicateRequiresAttribute), ProducedBy = ProducedBy.Trimmer)]
 		static void MethodWithDuplicateRequiresAttribute ()
 		{
 		}
 
+		[ExpectedWarning ("IL2026", "--MethodWithDuplicateRequiresAttribute--")]
 		[ExpectedWarning ("IL2026", "--MethodWithDuplicateRequiresAttribute--")]
```

Comment:
Could you modify the RUC message so that it's clear that one of the warnings comes from the attribute in code and the other from the attribute in XML?

Reply:
OK - got it